### ISSUE TITLE
False Positive `NoStringInterpolationWithoutBraces`

### ISSUE THREAD

**Issue by loganhenson** [tightenco/tlint#38]
```php
Cache::has("{$this->user_id}.accessed.{$slug}");
```
